### DISCUSSION TITLE
grant permission on external Android data

### DISCUSSION THREAD

**Discussion by john-peterson** [TeamAmaze/AmazeFileManager#4024] [Ideas]
I can grant access on internal Android data

No such request is made on external Android data

Is this intentionally or unintentionally intended

The result is that all read and write fail in app folders

It can still list all folders inside external Android data

**Comment by john-peterson** [comment]
I am talking about the last one here

```
/data/data no
/sdcard/Android/data yes
/storage/4A21-0000/Android ???

**Reply by john-peterson** [reply]
Actually the internal one doesn't ask for permission either

It was only in solid explorer

**Reply by john-peterson** [reply]
But solid doesn't even list anything inside external

**Comment by VishnuSanal** [comment]
@TranceLove :)
